cmd/internal/initializer/service.go

```diff
@@ -43,8 +44,9 @@ func (s *backupService) ListBackups(ctx context.Context, _ *v1.ListBackupsReques
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	// List internally sorts the backups
 	backups := versions.List()
-	versions.Sort(backups, false)
+	common.Sort(backups)
```

Comment:
Not needed for this PR but it would actually be nice to have a test that ensures that really the latest backup is getting restored if there are many.

Reply:
#91 created